src/pyodide/python-entrypoint-helper.ts

```diff
@@ -229,18 +229,17 @@ function makeEntrypointProxyHandler(
       if (typeof prop !== 'string') {
         return Reflect.get(target, prop, receiver);
       }
+      // Proxy calls to `fetch` to methods named `on_fetch` (and the same for other handlers.)
       const isKnownHandler = SUPPORTED_HANDLER_NAMES.includes(prop);
       if (isKnownHandler) {
         prop = 'on_' + prop;
       }
+
       return async function (...args: any[]): Promise<any> {
+        // Check if the requested method exists and if so, call it.
         const pyInstance = await pyInstancePromise;
         if (typeof pyInstance[prop] === 'function') {
-          const res = await doPyCall(pyInstance[prop], args);
```

Comment:
What's the reason for this change?

Reply:
Looks like something I missed in the rebase